data/categories.txt

```diff
@@ -5565,35 +5565,35 @@ sk:Oceán
 sw:Bahari
 
 @wifi
-en:WiFi
-ru:WiFi
+en:WiFi|Wi-Fi
+ru:WiFi|Wi-Fi
 ar:وايفاي
```

Comment:
So will it also work for Arabic, Japanese and Chinese, if this is not fixed? Or is that what is done in the `android/res/values/strings.xml` part? :)